### DISCUSSION TITLE
undiscord not working

### DISCUSSION THREAD

**Discussion by Morpheus777777** [victornpb/undiscord#396] [1. Questions & Answers]
So, I'm trying to mass delete a bunch of messages undiscord that are on certain servers. I was able to mass delete 10 to 50 messages on a couple servers today, but there are some servers that for some reason undiscord won't let me delete the messages on. I have no idea why, a few months ago I used undiscord to,delete thoussnds of messages so I know it has nothing to do with the number of messages. Also, this app still works perfectly fine for dms, and for some servers that I'm on, the only place where I can't actually delete my DMs are some servers in the general chat what I know for a fact that I have messages in the general chat that I don't want to have to hand delete one by one.

For futther information I'm going to provide you the error message and hopefully you guys can help me out.

Started at 7/10/2022, 1:05:17 AM

authorId="REDACTED" guildId="REDACTED" channelId="REDACTED" minId="REDACTED" maxId="REDACTED" hasLink=false hasFile=false

Error searching messages, API responded with status 403! 	{"message":"Missing Access","code":50001}

I did EVERYTHING right, like I had always done before and this is the first time I ever had an issue. I don't know why the hell I have to deal with this issue now but I really want help

**Comment by jsfag** [comment]
The same.
